packages/graphql-playground-react/src/state/sessions/sagas.ts

```diff
@@ -42,8 +42,7 @@ import * as queryString from 'query-string'
 function* setQueryFacts() {
   // debounce by 100 ms
   yield call(delay, 100)
-  const session: Session = yield select(getSelectedSession)
-
+  const session: Session = yield getSessionWithCredentials()
```

Comment:
This will include global headers (and credentials) on the schema fetch that could happen when text changes in the query window.  I'm wondering if there was a reason playground wasn't using `getSessionWithCredentials` here or if it was an oversight.